packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -460,6 +460,7 @@ const PipelineEditor = forwardRef(
           const [file] = await onFileRequested?.({
             canSelectMany: false,
             filters: { File: extensions },
+            property: "filename",
```

Comment:
Should this be `propertyID`?

Reply:
oh yes good catch

I should really add types to this function

Reply:
I ended up removing this property all together in this case